lambda-assets/things/update-thing-shadow/app.ts

```diff
@@ -0,0 +1,43 @@
+import { IoTDataPlaneClient, UpdateThingShadowCommand } from '@aws-sdk/client-iot-data-plane';
+import { Request, Response } from '@softchef/lambda-events';
+
+export async function handler(event: { [key: string]: any }) {
+  const request = new Request(event);
+  const response = new Response();
+  try {
+    const validated = request.validate(joi => {
+      return {
+        payload: joi.object().keys({
+          state: joi.object().keys({
+            desired: joi.object().required(),
```

Comment:
allow null

Reply:
payload is required in UpdateThingShadowCommand.

Reply:
desired & reported are optional